tests/helpers.py

```diff
@@ -1270,9 +1271,13 @@ def delete_objs_parallel(obj_list):
         bool: True if obj deleted else False
 
     """
-    with ThreadPoolExecutor() as p:
-        for obj in obj_list:
-            p.submit(obj.delete)
+    threads = list()
```

Comment:
@ramkiperiy wouldn't this resolve the issue of pvc going into terminating state? why should we use ignore leftover marker?

Reply:
During the testing we have ceph pod re-spin, due to this also observing leftovers.